independent-projects/bootstrap/maven-plugin/src/main/java/io/quarkus/maven/ExtensionDescriptorMojo.java

```diff
@@ -21,168 +21,235 @@
 import org.eclipse.aether.RepositorySystemSession;
 import org.eclipse.aether.repository.RemoteRepository;
 
-import com.eclipsesource.json.Json;
```

Comment:
Revert the formatting please. Or enable the Quarkus official one.

Reply:
seems like for some reason codeformatting is disabled on independent projects - when I run the codeformatter using ide-format the diff affects the following:

```
M core/src/main/java/io/quarkus/bootstrap/BootstrapClassLoaderFactory.java
 M core/src/main/java/io/quarkus/bootstrap/DefineClassVisibleURLClassLoader.java
 M core/src/main/java/io/quarkus/bootstrap/model/AppArtifactCoords.java
 M core/src/main/java/io/quarkus/bootstrap/model/AppArtifactKey.java
 M core/src/main/java/io/quarkus/bootstrap/model/AppDependency.java
 M core/src/main/java/io/quarkus/bootstrap/model/AppModel.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/AppModelResolver.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/BootstrapAppModelResolver.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/BootstrapWagonProvider.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/BuildDependencyGraphVisitor.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/DeploymentInjectingDependencyVisitor.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenArtifactResolver.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenLocalRepositoryManager.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenModelBuilder.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/MavenRepoInitializer.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/SecDispatcherImpl.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/SettingsDecrypterImpl.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/LocalProject.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/LocalWorkspace.java
 M core/src/main/java/io/quarkus/bootstrap/resolver/maven/workspace/ModelUtils.java
 M core/src/main/java/io/quarkus/bootstrap/util/BootstrapUtils.java
 M core/src/main/java/io/quarkus/bootstrap/util/IoUtils.java
 M core/src/main/java/io/quarkus/bootstrap/util/PropertyUtils.java
 M core/src/main/java/io/quarkus/bootstrap/util/ZipUtils.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/CollectDependenciesBase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/ResolverSetupCleanup.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/TsArtifact.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/TsDependency.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/TsJar.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/TsQuarkusExt.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/TsRepoBuilder.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/maven/test/MavenRepoInitializerTest.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/replace/test/SimpleReplacedDependencyTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DependenciesOnDifferentVersionsOfAnArtifactTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DependencyPresentTwiceInTheGraphInCompileAndProvidedScopesTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DependencyPresentTwiceInTheGraphInCompileAndTestScopesTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DependencyPresentTwiceInTheGraphTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DependencyPresentTwiceInTheGraphWithDifferentClassifierAndTypeTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DirectDependencyVersionOverridesTransitiveVersionTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/DirectOptionalOverridesTransitiveNonOptionalVersionTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/ExclusionsTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/OnlyDirectOptionalDepsAreCollectedTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/ProvidedScopeDepsAreNotCollectedTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/TestScopeIsNotAmongRuntimeDependenciesTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/resolver/test/TransitiveVersionOverridesTestCase.java
 M core/src/test/java/io/quarkus/bootstrap/util/ZipUtilsTest.java
 M core/src/test/java/io/quarkus/bootstrap/workspace/test/LocalWorkspaceDiscoveryTest.java
 M core/src/test/java/io/quarkus/bootstrap/workspace/test/MvnProjectBuilder.java
 M maven-plugin/src/main/java/io/quarkus/maven/AbstractTreeMojo.java
 M maven-plugin/src/main/java/io/quarkus/maven/ExtensionDescriptorMojo.java
 M maven-plugin/src/test/java/io/quarkus/maven/TreeMojoTestBase.java
```

For this PR I've re-formatted just the single file you asked for but we should get it enabled for the project automatically going forward.